test/e2e/generator/generate.go

```diff
@@ -140,9 +140,8 @@ func generateTestnet(r *rand.Rand, opt map[string]interface{}) (e2e.Manifest, er
 		GenesisCoreChainLockedHeight: uint32(opt["initialCoreChainLockedHeight"].(int)),
 		InitAppCoreChainLockedHeight: uint32(opt["initAppCoreChainLockedHeight"].(int)),
 		InitialState:                 opt["initialState"].(map[string]string),
-		Validators:                   &map[string]int64{},
+		Validators:                   map[string]int64{},
 		ValidatorUpdates:             map[string]map[string]int64{},
-		ChainLockUpdates:             map[string]int64{},
```

Comment:
why it's removed? we don't support it anymore?

Reply:
@QuantumExplorer disabled it [here](https://github.com/dashevo/tenderdash/blob/v0.8-dev/test/e2e/generator/generate.go#L227)
@QuantumExplorer should we uncomment this line?

Reply:
Most likely it was a mistake, and we should have the chainLockUpdates